services/minespace-web/src/styles/components/Forms.scss

```diff
@@ -13,6 +13,15 @@
     }
 }
 
+// .ant-form-item-checkbox-required .ant-checkbox+span {
+//     &::after {
+//         content: "*";
+//         font-family: $font-family;
+//         color: $gov-red;
+//     }
+
+// }
```

Comment:
Will this commented code still be used in future ?

Reply:
ah this got missed in my reverting/unreverting. Going to re-instate it- it's necessary to show required mark on checkbox. Good catch!